### PR TITLE
Adding Asmr_Racoon ref files

### PR DESCRIPTION
**Pull request by DrewThomasson**

### CODE REVIEW

**Comment by DrewThomasson** [comment]
Generated Example from voice

https://github.com/user-attachments/assets/ef95036d-19ff-4c9f-9650-30735f549505

**Comment by ROBERT-MCDOWELL** [comment]
be careful, too much noise in background. you must use a noise gate at least, then upload it as a cloned voice to profit from the audio treatment and use the result for your fine tuned.

**Comment by DrewThomasson** [comment]
to train it?

**Comment by ROBERT-MCDOWELL** [comment]
yes

**Comment by DrewThomasson** [comment]
oh I tried but there was a ticking sound which messed up any de-noising attempts :/

**Comment by ROBERT-MCDOWELL** [comment]
a ticking sound after you uploaded the original audio voice as a cloned voice to gradio?

**Comment by DrewThomasson** [comment]
No like in the training data

, We don't have too use this voice btw I have other tts fine-tune failures

**Comment by DrewThomasson** [comment]
Unless I can....

one sec

**Comment by ROBERT-MCDOWELL** [comment]
as upload clone voice function cut the audio at max 12 sec so maybe for all your audio data for finetuned it needs you use the normalization tool script in tools.... ref.wav should be also treated.

**Comment by DrewThomasson** [comment]
I'm testing to see if treating the ref.wav is enough... if not then 🤷

**Comment by DrewThomasson** [comment]
Thankfully the ref is only 10 sec long

**Comment by DrewThomasson** [comment]
How's this?
[RacoonASMRDenoisedRef.wav.zip](https://github.com/user-attachments/files/20577881/RacoonASMRDenoisedRef.wav.zip)

# Generated Example

https://github.com/user-attachments/assets/66733da4-57f0-4765-9f15-786ccd593e3b

**Comment by ROBERT-MCDOWELL** [comment]
perfect ;)

**Comment by DrewThomasson** [comment]
:D

**Comment by DrewThomasson** [comment]
kk I merge then :)
